### PR TITLE
#167221908 Deploy Base(staging) Application To Heroku 

### PR DESCRIPTION
**Pull request by GerrardE**
#### What does this PR do?
Set Up Heroku Deploy

#### Description of Task to be completed?
- Configure repo static and app json files
- Create kingsmen-ah-frontend pipeline via heroku
- Create kingsmen-ah-frontend-staging application
- Enable review apps

#### How should this be manually tested?
- Visit https://kingsmen-ah-frontend-staging.herokuapp.com/ to see the deployed application

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#167221908](https://www.pivotaltracker.com/story/show/167221908)

#### Screenshots (if appropriate)
Screenshot 2019-07-12 at 7 25 25 AM

#### Questions:
N/A

### CODE REVIEW

**Review by adex001 — CHANGES_REQUESTED** [review]
Nice job! implement the changes requested and you should be fine!

File: package.json [hunk 1]
```diff
@@ -100,5 +100,9 @@
     "webpack": "^4.35.2",
     "webpack-cli": "^3.3.5",
     "webpack-merge": "^4.2.1"
+  },
+  "engines": {
+    "node": "10.15.3",
+    "npm": "6.10.0"
   }
-}
+}
```
Comment (adex001) [inline package.json]: add a new line

**Review by adex001 — DISMISSED** [review]
Great work!

**Review by despeauxz — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (fidelisojeah) [reply package.json]: @GerrardE are you going to fix this?

**Review by fidelisojeah — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (GerrardE) [reply package.json]: I have fixed it. Thanks

**Review by adex001 — APPROVED** [review]

**Review by fidelisojeah — APPROVED** [review]

**Pull request merged**